src/context/AlertContext.tsx

```diff
@@ -0,0 +1,27 @@
+import { createContext, useContext, useState } from 'react';
+
+type AlertContextType = {
+  isAlertVisible: boolean;
+  setIsAlertVisible: Dispatch<SetStateAction<boolean>>;
+};
+
+const AlertContext: Context<AlertContextType> = createContext<AlertContextType>({
+  isAlertVisible: false,
+  setIsAlertVisible: () => {},
+});
+
+export const AlertProvider = ({ children }: JSX.Element): JSX.Element => {
+  const [isAlertVisible, setIsAlertVisible] = useState(false);
+
+  return (
+    <AlertContext.Provider valaue={{ isAlertVisible, setIsAlertVisible }}>
```

Comment:
```suggestion
    <AlertContext.Provider value={{ isAlertVisible, setIsAlertVisible }}>
```

Reply:
ah, this could be typo of the year haha

updated in https://github.com/RR-Ardhika/murojaah/pull/7/commits/a4ba87dbc135213428b4000a6f047d111d4a6f29, thank you